servant-rate-limit/servant-rate-limit.cabal

```diff
@@ -119,9 +119,6 @@ test-suite servant-rate-limit-tests
   if flag(client)
     build-depends:
         servant-client
-  if flag(openapi)
```

Comment:
Not sure where this comes from, but this caused build errors with cabal:

```
servant-rate-limit/servant-rate-limit.cabal:0:0: error:
These flags are used without having been defined: openapi
```

running `hpack` in that directory resolved it.

Reply:
It leaked out from the work I was doing for #9 it seems. I am surprised that didn't cause any problems in CI that caught this sooner to be honest. I think I will rewrite that commit, since it's unreleased.